src/CompanyService.Models.Db/DbDepartment.cs

```diff
@@ -37,7 +37,8 @@ public void Configure(EntityTypeBuilder<DbDepartment> builder)
 
             builder
                 .HasMany(d => d.Users)
-                .WithOne(u => u.Department);
+                .WithOne(u => u.Department)
+                .HasForeignKey(k => k.DepartmentId);
```

Comment:
this key have to be in DepartmentUser not here. DbDepartment is main entity, DepartmentUser dependent

Reply:
done